#! /bin/sh /usr/share/dpatch/dpatch-run
## 20-hide-help-buttons.dpatch by Felix Geyer <debfx-pkg@fobos.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Hide useless help buttons in dialogs

@DPATCH@
diff -urNad virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxCloseVMDlg.ui virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxCloseVMDlg.ui
--- virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxCloseVMDlg.ui	2009-10-07 11:27:13.000000000 +0200
+++ virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxCloseVMDlg.ui	2009-10-07 13:44:29.935463062 +0200
@@ -220,7 +220,7 @@
    <item>
     <widget class="QIDialogButtonBox" name="mButtonBox" >
      <property name="standardButtons" >
-      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
+      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
diff -urNad virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxMediaManagerDlg.ui virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxMediaManagerDlg.ui
--- virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxMediaManagerDlg.ui	2009-10-07 11:27:13.000000000 +0200
+++ virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxMediaManagerDlg.ui	2009-10-07 13:44:29.935463062 +0200
@@ -348,7 +348,7 @@
       <item>
        <widget class="QIDialogButtonBox" name="mButtonBox" >
         <property name="standardButtons" >
-         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
+         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
diff -urNad virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxSettingsDialog.ui virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxSettingsDialog.ui
--- virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxSettingsDialog.ui	2009-10-07 11:27:13.000000000 +0200
+++ virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxSettingsDialog.ui	2009-10-07 13:44:29.935463062 +0200
@@ -145,7 +145,7 @@
        <item row="3" column="0" colspan="2" >
         <widget class="QIDialogButtonBox" name="mButtonBox" >
          <property name="standardButtons" >
-          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
+          <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
diff -urNad virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxSnapshotDetailsDlg.ui virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxSnapshotDetailsDlg.ui
--- virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxSnapshotDetailsDlg.ui	2009-10-07 11:27:13.000000000 +0200
+++ virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxSnapshotDetailsDlg.ui	2009-10-07 13:44:29.935463062 +0200
@@ -126,7 +126,7 @@
    <item>
     <widget class="QIDialogButtonBox" name="mButtonBox" >
      <property name="standardButtons" >
-      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
+      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
diff -urNad virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxTakeSnapshotDlg.ui virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxTakeSnapshotDlg.ui
--- virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxTakeSnapshotDlg.ui	2009-10-07 11:27:13.000000000 +0200
+++ virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxTakeSnapshotDlg.ui	2009-10-07 13:44:29.935463062 +0200
@@ -114,7 +114,7 @@
    <item>
     <widget class="QIDialogButtonBox" name="mButtonBox" >
      <property name="standardButtons" >
-      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
+      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
diff -urNad virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxVMLogViewer.ui virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxVMLogViewer.ui
--- virtualbox-ose-3.0.8-dfsg~/src/VBox/Frontends/VirtualBox/ui/VBoxVMLogViewer.ui	2009-10-07 11:27:13.000000000 +0200
+++ virtualbox-ose-3.0.8-dfsg/src/VBox/Frontends/VirtualBox/ui/VBoxVMLogViewer.ui	2009-10-07 13:44:29.935463062 +0200
@@ -44,7 +44,7 @@
     <item>
      <widget class="QIDialogButtonBox" name="mButtonBox" >
       <property name="standardButtons" >
-       <set>QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Save</set>
+       <set>QDialogButtonBox::Close|QDialogButtonBox::NoButton|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
